config/settings.yml

```diff
@@ -980,6 +980,7 @@
   :name:
   :maindb: ExtManagementSystem
   :container_deployment_wizard: false
+  :run_automate_methods_on_service_catalog_api_submit: false
```

Comment:
The only thing I would suggest is dropping the `catalog` from the name.  Technically it is a `service_template` but I think `run_automate_methods_on_service_api_submit` is enough.